samples/cpp/continuous_batching_benchmark/continuous_batching_benchmark.cpp

```diff
@@ -123,6 +121,11 @@ Dataset filtered_dataset(const std::string& models_path, const std::string& data
 
         ov::genai::GenerationConfig greedy_search = ov::genai::greedy();
         greedy_search.max_new_tokens = std::min(max_output_len, output_len);
+        greedy_search.repetition_penalty = 1.0;
+        greedy_search.frequency_penalty = 0.0;
+        greedy_search.presence_penalty = 0.0;
+        greedy_search.diversity_penalty = 0.0;
+        greedy_search.length_penalty = 0.0;
```

Comment:
by default all penalties should be OFF (if it's not a case, we need to fix it), so do we need to set them explicitly here?

Reply:
I suppose they were not OFF at a time this change was made. I agree we should rather change defaults than set it to 0 here.

Reply:
Could you please set then to 0 if it's not done yet? 
And update this place accordingly